docs/PnPDeploySpArtifacts.md

```diff
@@ -0,0 +1,145 @@
+---
```

Comment:
@baywet as discussed here:
https://github.com/pnp/sp-alm-devops-extension/pull/1#discussion_r274410698

I added documentation for it. Do you think that this is sufficient?

Reply:
yeah I think it's perfect!